### ISSUE TITLE
Svelte client: couldn't create connection to server.

### ISSUE THREAD

**Issue by RostykV1** [sveltejs/language-tools#2181]
### Describe the bug

While opening VSCODE, receiving: 
[Error - 7:38:34 PM] Svelte client: couldn't create connection to server.
Launching server using runtime velteserver --stdio --compiler-warnings a11y-autofocus:ignore,a11y-no-onchange:ignor failed. Error: spawn velteserver --stdio --compiler-warnings a11y-autofocus:ignore,a11y-no-onchange:ignor ENOENT

### Reproduction

None

### Expected behaviour

None

### System Info

- OS: WINDOWS 10 PRO
- IDE: VSCODE

### Which package is the issue about?

Svelte for VS Code extension

### Additional Information, eg. Screenshots

![image](https://github.com/sveltejs/language-tools/assets/80162271/0cf1af11-1af4-4d84-84c9-583ed0f41bdd)

**Comment by jasonlyu123** [comment]
Please read the config description before copying a random script into the config. [svelte.language-server.runtime](https://github.com/sveltejs/language-tools/tree/master/packages/svelte-vscode#sveltelanguage-serverruntime) config is the path to your node.js installation. It can not be a command to launch your language server. The [svelte.language-server.runtime](https://github.com/sveltejs/language-tools/tree/master/packages/svelte-vscode#sveltelanguage-serverruntime) config is for a custom language server but should be the path to the language server js file. It also can't be a command.
